views/js/runner/plugins/navigation/previous.js

```diff
@@ -66,15 +79,46 @@ define([
             }));
 
             //attach behavior
-            this.$element.on('click', function(e){
-                e.preventDefault();
-                if(self.getState('enabled') !== false){
-                    self.disable();
+            function doPrevious(previousItemWarning) {
+                var context = testRunner.getTestContext();
 
-                    testRunner.previous();
+                previousItemWarning = previousItemWarning || false;
+
+                function enable() {
+                    testRunner.trigger('enablenav enabletools');
                 }
+
+                if(self.getState('enabled') !== false){
+                    testRunner.trigger('disablenav disabletools');
+
+                    if (previousItemWarning && context.remainingAttempts !== -1) {
+                        testRunner.trigger(
+                            'confirm.previous',
+                            __('You are about to go to the previous item. Click OK to continue and go to the previous item.'),
```

Comment:
The message is redundant...
What about something like:
"You are about to go to the previous item. You may not be able to go back to the current item. Continue anyway ?"